### PR TITLE
Variable -> Tensor

### PR DESCRIPTION
**Pull request by zhupengyang**
image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: doc/fluid/api_cn/layers_cn/uniform_random_cn.rst [hunk 1]
```diff
@@ -19,18 +19,19 @@ uniform_random
...
 
-返回：表示一个随机初始化结果的Tensor，该Tensor的数据类型由dtype参数决定，该Tensor的维度由shape参数决定。
-    
-返回类型：Variable
+返回：
+    Tensor：数值服从范围[``min``, ``max``)内均匀分布的随机Tensor，形状为 ``shape``，数据类型为 ``dtype``。
 
 抛出异常：
-    - :code:`TypeError`: shape的类型应该是list、tuple 或 Variable。
+    - ``TypeError`` - 如果 ``shape`` 的类型不是list、tuple、Variable。
```
Comment (lanxianghit) [inline doc/fluid/api_cn/layers_cn/uniform_random_cn.rst]:
```suggestion
    - ``TypeError`` - 如果 ``shape`` 的类型不是list、tuple、Tensor。
```

File: doc/fluid/api_cn/layers_cn/gaussian_random_cn.rst [hunk 2]
```diff
@@ -3,30 +3,29 @@
...
+    - **dtype** (str|np.dtype|core.VarDesc.VarType, 可选) - 输出Tensor的数据类型，支持float32、float64。默认值为float32。
+    - **name** (str, 可选) - 输出的名字。一般无需设置，默认值为None。该参数供开发人员打印调试信息时使用，具体用法请参见 :ref:`api_guide_Name` 。
 
-返回：        
+返回：
+    Tensor：符合高斯随机分布的Tensor，形状为 ``shape``，数据类型为 ``dtype``。
 
-        - 符合高斯分布的随机 Tensor。形状为 shape，数据类型为 dtype。
+抛出异常：
+  - ``TypeError`` - 如果 ``shape`` 的类型不是list、tuple、Variable。
```
Comment (lanxianghit) [inline doc/fluid/api_cn/layers_cn/gaussian_random_cn.rst]:
```suggestion
  - ``TypeError`` - 如果 ``shape`` 的类型不是list、tuple、Tensor。
```

File: doc/fluid/api_cn/layers_cn/uniform_random_cn.rst (hunk 1 above)
Comment (zhupengyang) [reply doc/fluid/api_cn/layers_cn/uniform_random_cn.rst]: Done

File: doc/fluid/api_cn/layers_cn/gaussian_random_cn.rst (hunk 2 above)
Comment (zhupengyang) [reply doc/fluid/api_cn/layers_cn/gaussian_random_cn.rst]: Done

**Review by lanxianghit — APPROVED** [review]
LGTM

**Review by swtkiwi — APPROVED** [review]
LGTM

**Pull request merged**